examples/blog-update/overrides/partials/post.html

```diff
@@ -0,0 +1,67 @@
+{#-
+  This file was automatically generated - do not edit
+-#}
+<article class="md-post md-post--excerpt">
+  <header class="md-post__header">
+    {% if post.authors %}
+      <nav class="md-post__authors md-typeset">
+        {% for author in post.authors %}
+          <span class="md-author">
+            <img src="{{ author.avatar }}" alt="{{ author.name }}">
+          </span>
+        {% endfor %}
+      </nav>
+    {% endif %}
+    <div class="md-post__meta md-meta">
+      <ul class="md-meta__list">
+        <li class="md-meta__item">
+          <time datetime="{{ post.config.date.created }}">
+            {{- post.config.date.created | date -}}
+          </time>
+          <!-- start customization -->
+          {% if post.config.date.updated %}
+          <time datetime="{{ post.config.date.updated }}">
+            &nbsp;({{- post.config.date.updated | date -}})
```

Comment:
That is actually a smell – `&nbsp` should not be necessary. If you remove the `-` in the placeholder you should be able to remove it, as those are used for whitespace control.

Reply:
I do need to study some Jinja... will do it just after my 100 pushups ;o)